src/compas_fab/backends/kinematics/analytical_inverse_kinematics.py

```diff
@@ -0,0 +1,143 @@
+from compas.geometry import Frame
+from compas.robots import Configuration
+from compas_fab.backends.exceptions import BackendError
+from compas_fab.backends.kinematics.utils import fit_within_bounds
+from compas_fab.backends.interfaces import InverseKinematics
+
+
+class Analytical_InverseKinematics(InverseKinematics):
```

Comment:
I would remove the underscore from this class name, ie `AnalyticalInverseKinematics`